run-tests.php

```diff
@@ -664,6 +666,11 @@ function main()
         }
     }
 
+    if ($selected_tests && count($test_files) === 0) {
+        echo 'No tests found.' . PHP_EOL;
+        exit;
```

Comment:
```suggestion
        echo "No tests found.\n";
        return;
```

Going by the majority of printing happening in this script, we don't use PHP_EOL.